src/index.html

```diff
@@ -13,12 +13,14 @@
     </head>
     <body>
         <nav class="navigation">
+            <img src="/src/img/Logo.png" class="navigation__logo" alt="Doctor Care logo">
```

Comment:
why is the image inside the navigation? it should be part of the header and not the navigation. Only things enter the navigation that make sense, that when clicked lead to another corner.

Fix it!